android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -440,9 +440,10 @@ public void onDataFetched(JSONObject jsonObject, BranchError error) {
     }
 
     @ReactMethod
-    public void setIdentity(String identity) {
+    public void setIdentity(String identity, Promise promise) {
         Branch branch = Branch.getInstance();
         branch.setIdentity(identity);
```

Comment:
Do we need to pass in anything for the second argument `BranchReferralInitListener` ?